material-overrides/assets/stylesheets/polkadot.css

```diff
@@ -982,6 +982,10 @@ input.md-search__input::placeholder {
   }
 }
 
+article.md-content__inner.md-typeset > h1 {
```

Comment:
Not in love with this for pages that don't have badges. I think we should apply the styling to the badge itself